examples/chef/common/chef-rvc-operational-state-delegate.cpp

```diff
@@ -17,17 +17,31 @@
  */
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <app/util/config.h>
-
-#ifdef ZCL_USING_OPERATIONAL_STATE_RVC_CLUSTER_SERVER
 #include <chef-rvc-operational-state-delegate.h>
+#include <platform/CHIPDeviceLayer.h>
 
 using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::OperationalState;
 using namespace chip::app::Clusters::RvcOperationalState;
+using chip::Protocols::InteractionModel::Status;
+
+#ifdef MATTER_DM_PLUGIN_RVC_OPERATIONAL_STATE_SERVER
```

Comment:
why was this moved?

Why do we ifdef the entire file? can we instead use the build system to either include the cpp file or not?

Reply:
moved back

Reply:
right. it's not necessary as includes already specified in stubs.cpp. removed

Reply:
actually it caused dimmablelight build failed so i'm adding it back